lib/plugins.ts

```diff
@@ -142,14 +142,14 @@ export interface ProtractorPlugin {
    * @throws {*} If this function throws an error, a failed assertion is added to
    *     the test results.
    *
-   * @return {Q.Promise=} Can return a promise, in which case protractor will wait
-   *     for the promise to resolve before continuing.  If the promise is
-   *     rejected, a failed assertion is added to the test results, and protractor
-   *     will continue onto the next command.  If nothing is returned or something
-   *     other than a promise is returned, protractor will continue onto the next
-   *     command.
+   * @return {webdriver.promise.Promise|Q.Promise} Can return a promise, in which
+   *     case protractor will wait for the promise to resolve before continuing. If
+   *     the promise is rejected, a failed assertion is added to the test results,
+   *     and protractor will continue onto the next command. If nothing is returned
+   *     or something other than a promise is returned, protractor will continue
+   *     onto the next command.
    */
-  waitForPromise?: () => q.Promise<any>;
+  waitForPromise?(): webdriver.promise.Promise<void>|q.Promise<void>;
```

Comment:
`q.Promise` is actually not allowed here - only webdriver's promise implementation.  Could you fix the type/comment?